### PR TITLE
Add SiteMode.inline_css option for admins to override styles on the fly

### PR DESCRIPTION
**Pull request by ejucovy**
I wrote this to depend on #191 (in migration order) but can redo it if there ends up being a reason not to merge #191 first.

Do you think this merits a test?

Also debated adding validation (e.g. with tinycss) so that admins have confidence they won't break things with a typo'ed css line (or, worse, a stray `</style></head><body></body></html>` in their css content...) but felt it's probably not worth doing now.

### CODE REVIEW

File: opendebates/models.py [hunk 1]
```diff
@@ -42,6 +42,8 @@ class SiteMode(CachingMixin, models.Model):
     )
     debate_state = models.CharField(max_length=5, null=True, blank=True)
 
+    inline_css = models.TextField(null=True, blank=True)
```
Comment (tobiasmcnulty) [inline opendebates/models.py]: Same comment about `null=True` (unless there's a reason the project needs nullable text/char fields?)

**Comment by tobiasmcnulty** [comment]
Looks good to me with a couple comments:
- It's probably worth writing a simple test just to make sure we don't inadvertently lose the functionality down the road.
- If the primary purpose of this ATM is to hide the header logos, would it be worthwhile to add IDs to those (or even convert them to elements with the logo as a `background-image`)? I'm struggling to come up with some CSS that can easily hide them without that.

**Comment by ejucovy** [comment]
Good idea.  I was going to send Lilia something involving `header img:not(:first-child) { visibility: hidden }` but that's unreasonable.  I've added `partner-logo` and `partner-logo-N` classes to each -- there are two copies of each logo, one for desktop and one for mobile, so this should also make it easy for future admins to e.g. swap in a different partner-logo-2 everywhere with one line of css.

**Comment by ejucovy** [comment]
> (or even convert them to elements with the logo as a background-image)

-- er, right, I forgot these are ``s and not things with backgrounds.  I don't think I have time (and the courage, given our launch timeline) to get that change done and browser-tested right now, but you're right that they probably ought to be converted eventually for more flexibility.

In the short term if Lilia ends up needing to actually replace these images on the fly (vs just toggling their existence) I'll show her how to do it with cloudflare rules, which was the hack I came up with when we needed to do it at the last minute last time around. :-)

**Comment by tobiasmcnulty** [comment]
do you mind re-syncing this branch (maybe merge develop?) to see if you can get GitHub to stop showing the phone number changes?

**Comment by tobiasmcnulty** [comment]
That did it thanks!

File: opendebates/tests/test_site_mode.py [hunk 2]
```diff
@@ -86,3 +86,43 @@ def test_announcement_page_regex(self):
         rsp = self.client.get('/')
         self.assertIn('<div class="site-announcement warning">', rsp.content)
         self.assertIn('<a href="%s">' % link, rsp.content)
+
+
+class InlineCSSTest(TestCase):
+
+    def setUp(self):
+        self.mode, _ = SiteMode.objects.get_or_create()
+        self.mode.save()
```
Comment (tobiasmcnulty) [inline opendebates/tests/test_site_mode.py]: Is this needed (`mode.save()`)?

Comment (ejucovy) [reply opendebates/tests/test_site_mode.py]: Nope!

**Comment by tobiasmcnulty** [comment]
🚀

**Pull request merged**